doc/extending/op.txt

```diff
@@ -598,6 +631,7 @@ Trying out our new Op
 
 In the following code, we use our new Op:
 
+>>> import theano
```

Comment:
You would need to add `.. doctest:: mul` to make the doctest block part of mul.

Same with the other doctest block below.